src/redux/features/greetingSlice.js

```diff
@@ -0,0 +1,37 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  message: '',
+  loading: false,
+};
+
+export const fetchMessage = createAsyncThunk(
+  'greeting/fetchMessage',
+  async () => {
+    const response = await fetch(
+      'http://localhost:3000/message/index',
```

Comment:
- [ ] Your two apps are not successfully connecting together. When I open `http://localhost:3000/message/index` on the browser, I got the error on the screenshot below:
![Screenshot from 2023-10-08 05-35-00](https://github.com/yosaddis/hello-react-front-end/assets/63087491/0783434c-c5d0-423d-8f16-402f244583df)

Reply:
![image](https://github.com/yosaddis/hello-react-front-end/assets/60548252/03ca33d4-68ea-4941-b178-f60dc65bc343)
@FrederickMih Thanks for the feedback. I checked on my side and got this. Any suggestions on where I look?